client/components/domains/domain-product-price/index.jsx

```diff
@@ -62,9 +63,17 @@ class DomainProductPrice extends React.Component {
 	}
 
 	renderIncludedInPremium() {
-		const textLabel = config.isEnabled( 'domains/kracken-ui' )
-			? this.props.translate( 'Included in Paid Plans' )
-			: this.props.translate( 'Included in WordPress.com Premium' );
+		const { translate, showExpandedPrice } = this.props;
+
+		let textLabel = this.props.translate( 'Included in WordPress.com Premium' );
+
+		const isKrackenUI = config.isEnabled( 'domains/kracken-ui' );
+		if ( isKrackenUI ) {
+			textLabel = showExpandedPrice
+				? translate( 'Price included in paid plans' )
```

Comment:
Probably because you're used to it ;)

I personally prefer the shorter copy too, both aesthetically and in code. Nonetheless, I assume we'd want to stick with the longer copy here - @danhauk, can you confirm?